libs/application/templates/operating-license/src/lib/constants.ts

```diff
@@ -0,0 +1,145 @@
+import { DefaultEvents, Option } from '@island.is/application/types'
+
+import { attachmentNames, m } from './messages'
+
+export type Events =
+  | { type: DefaultEvents.SUBMIT }
+  | { type: DefaultEvents.PAYMENT }
+
+export enum States {
+  PREREQUISITES = 'prerequistites',
+  DRAFT = 'draft',
+  DONE = 'done',
+  PAYMENT = 'payment',
+}
+export enum Roles {
+  APPLICANT = 'applicant',
+}
+
+export enum ApiActions {
+  submitApplication = 'submitApplication',
+  createCharge = 'createCharge',
+  submitPassportApplication = 'submitPassportApplication',
+}
+
+export enum APPLICATION_TYPES {
+  HOTEL = 'hotel',
+  RESTURANT = 'resturant',
+}
+
+export const YES = 'yes'
+export const NO = 'no'
+
+export const ResturantTypes: Option[] = [
```

Comment:
wondering if these should be kept in messages for the translation purposes? or nah?

Reply:
I'll have to ask about it, not sure if these translations exist.. I'll do it in a follow up PR if needed